photography-site-demo.js/.env.example

```diff
@@ -0,0 +1,25 @@
+#Environment to choose. Possible values: astra, jsonapi
```

Comment:
Same comments as #23, let's avoid using non-standard NODE_ENV values. NODE_ENV is typically set to "local", "development", or "production". I would prefer to use an IS_ASTRA environment variable to determine whether connecting to Astra or JSON API.

Reply:
Thanks! Solved.